heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,25 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean consolePrompt(int oldContainerCount, int newContainerCount) {
+    String fmt =
+        "The present aurora job has %d containers. After update there will be %d containers. ";
+    Console c = System.console();
+    if (c == null) {
+      LOG.warning("No console to prompt user");
+      System.err.println(String.format(fmt, oldContainerCount, newContainerCount));
+      return true;
+    }
+
+    String userInput = c.readLine(
+        String.format(fmt + " Please make sure there are sufficient resources to update this job."
+            + " Continue update? [Y/n]: ", oldContainerCount, newContainerCount));
+    if ("n".equalsIgnoreCase(userInput)) {
```

Comment:
I think it might be better to check "y" instead of "n". don't forget to update "[Y/n]" to "[y/N]"

Reply:
updated